nest/04.finest.md

```diff
@@ -0,0 +1,191 @@
+
+When removing a requirement, `finest` makes a copy of your current dependency graph (if a copy wasn't already made), removes that requirement from its parent and removes its depedencies recursively if possible.
+
+Its output is similar to this:
+
+```
+$ finest requirement remove linux --parent @work
+Removing requirement stable::kernel/linux#* to group @work...
+```
+
+#### Common issues
+* `error: no package found for requirement <req>`
+
+This error means that `finest` couldn't find any package to satisfy this requirement.
+What probably went wrong, is that the syntax of the requirement is incorrect.
+You can check on [Raven-OS website](https://stable.raven-os.org/) to check the repository and the category of this requirement.
+
+### 4.2.2 Updating the system by merging the depedency graph
+#### Merging the dependency graph
+`finest merge` merges the copy of your dependency graph, on which `finest` applied the transactions, to your current dependency graph.
+These two graphs are compared to extract a list of transactions that `finest` will perform, in order to go from one graph to another.
+And so, your dependency graph will finally apply the transactions you meant to do.
+
+Example:
+```
+2 pending transactions:
+
+ install stable::kernel/linux#4.14.96
+ remove stable::sys-libs/glibc#2.0.1
+
+Would you like to apply these transactions? [Yes/no] Y
+Downloading stable::kernel/linux#4.14.96...
+Successfully installed stable::kernel/linux#4.14.96
+Removing stable::sys-libs/glibc#2.0.1...
+Successfully uninstalled stable::sys-libs/glibc#2.0.1
+```
+
+> Important
+>
+> Never forget to run `finest merge` after you're done with your requirements, or they won't be applied on the system
+
+### 4.2.3 Managing groups
+#### Create a group
+Creating a group is possible via the command `finest group create <name> [--parent <group>]`.
+It's very important to keep in mind that a group name always starts with a `@` character.
+By default, this will create a group as a child of the group `@root` if no parent group is provided.
+
+Example:
+```
+$ finest group add @groupName
+Adding group @groupName with parent group @root...
+Successfully added all the specified groups.
+```
+
+#### Delete a group
+In the same manner as creating a group, you can delete one with the command `finest group remove <name> [--parent <group>]`.
+
+Example:
+```
+$ finest group remove @WorkGroup1
+Removing group @WorkGroup1...
+Successfully removed all the specified groups.
+```
+
+#### List the groups
+To list all the groups on the OS, use `finest group list`.
+
+Example:
+```
+$ finest group list
+@root
+@WorkGroup2
+@SuperCoolProject
+```
+
+## 4.3 Summary
+This concludes the part of this getting-started dedicated to `finest`.
+What's left after this is a cheat sheet summarizing all the commands of both Nest binaries, and a glossary explaining the concepts and commands mentioned throughout this getting-started.
```

Comment:
Sounds to informal.

Reply:
I'm not sure if this is any better, but it's another way to write it at least
```suggestion
In the next chapter, you'll find a cheat sheet summarizing all the commands of `nest` and `finest`.
After it is a glossary explaining the concepts and commands mentioned throughout this getting-started, with links to their respective chapters or sections.
```

Reply:
up